docs/ref/contrib/postgres/aggregates.txt

```diff
@@ -349,16 +359,19 @@ field or an expression returning a numeric data. Both are required.
 Usage examples
 ==============
 
-We will use this example table::
+We will use this example table:
+
+.. code-block:: text
```

Comment:
```suggestion
.. code-block:: markdown
```
Could it be markdown ?

Reply:
Meh. Argh! — I'd like to punt on that to a separate cleanup please. (IDK 🤯)

Reply:
It was only a question. I agree with you.